Frameworks/UIKit/StarboardXaml/StarboardXaml.h

```diff
@@ -58,7 +58,7 @@ ref class AppEventListener
 
 extern "C" void _ApplicationLaunch(ActivationType activationType, Platform::Object^ activationArg);
 extern "C" void EbrApplicationActivated(Windows::ApplicationModel::Activation::IActivatedEventArgs^ args);
```

Comment:
The old activation path would be fine, and the new one will already need to reimport to launch

---
In reply to: [72338427](https://github.com/Microsoft/WinObjC/pull/686#discussion_r72338427) [](ancestors = 72338427,72336614)